### PR TITLE
Update matrixcoder.json

### PR DESCRIPTION
**Pull request by MatrixCoder0101**
Update Recodes

### CODE REVIEW

File: domains/matrixcoder.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
         "email": "tkgoutamm625@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
+        "CNAME": "matrixcoder0101.github.io"
```
Comment (DEV-DIBSTER) [inline domains/matrixcoder.json:7]: Cannot have CNAME in conjunction with other records.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/matrixcoder.json (hunk 1 above)
Comment (MatrixCoder0101) [reply domains/matrixcoder.json:9]: > Cannot have CNAME in conjunction with other records.

Sir any other way to use this domain in both GitHub Pages and ImproveMX email server ?

Comment (DEV-DIBSTER) [reply domains/matrixcoder.json:9]: > > Cannot have CNAME in conjunction with other records.
> 
> Sir any other way to use this domain in both GitHub Pages and ImproveMX email server ?

I believe GitHub allows you to do A records, you would have search in the repository settings. And then add your ImproveMX email settings which I assume are TXT and MX records normally. CNAME records just don't allow anything else.

Comment (MatrixCoder0101) [reply domains/matrixcoder.json]: Now check Is this are record correct?

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**